### ISSUE TITLE
Consider integrating with https://pypi.org/project/stringcontext/

### ISSUE THREAD

**Issue by fhightower** [democritus-project/d8s-strings#8]
**WHAT I want this project to do (or not do)**

Integrate with https://pypi.org/project/stringcontext/

**WHY I want this project to have this feature**

To improve our ability to view a string in context.

**Comment by raprocks** [comment]
Will it be okay to add this functionality as a function? this fairly simple to implement if we take care of the edge cases like if the context before and after is out of range such that the string is smaller and so others

**Comment by fhightower** [comment]
Hi @raprocks ! Thanks for asking. This issue was an old one that I moved from another place. I don't think we should work on it as the [stringcontext](https://pypi.org/project/stringcontext/) library looks pretty solid on its own and we don't want to duplicate functionality or needlessly wrap functions.

This is part of a larger discussion on what this library *is* (which is something I'm still thinking about and scoping), but in the past I have made the mistake of replicating other libraries or needlessly wrapping functions (there is an example that still exists [here](https://github.com/democritus-project/d8s-strings/blob/cd74ba0b7fa0fcb27d3da71a7a1bc95f98f0608d/d8s_strings/strings.py#L459)). If there is already a decent library to do something and there are no ways we can make the interface or functionality better, I'd say we shouldn't do anything. As such, I think we should close this issue and label it `wontfix`.

Thoughts?
